src/models/LandRequest.js

```diff
@@ -2,6 +2,11 @@
 import Joi from 'joi';
 import BaseModel from './BaseModel';
 
+/**
```

Comment:
Left this in for reference/comment. I started trying to add the Joi stuff but it was taking a lot of time for very little gain. Happy to look again later though if you want it in.